Anvil/Tools/ScanCore.pm

```diff
@@ -1654,6 +1666,13 @@ sub post_scan_analysis_node
 				$anvil->Alert->register({alert_level => "warning", message => "warning_0082", set_by => "ScanCore", variables => $variables});
 				$anvil->Email->send_alerts();
 				
+				if (!$action_log_disabled) {
```

Comment:
For formatting consistence and readability; We use: 

if (not $foo)
{
    ....
}

Reply:
Go that